### PR TITLE
feat(import): Add FlatFile Portal to ASSESS

### PR DESCRIPTION
**Pull request by vkozynskyi-extenda**

### CODE REVIEW

File: radar/dev/flatfileportal.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+name: Flatfile Portal
+blip:
+  - date: 2020-06-26
+    ring: ASSESS
+description: |
+  An add-on for web-based applications to allow customers or internal users to upload spreadsheets of data.
+  It pairs with SaaS dashboard and AI-assisted field match feature.
+rationale: |
+  HiiRetail needs reach UI capabilities, easy to embed and use, to allow users upload csv/xls data,
```
Comment (Tasseri) [inline radar/dev/flatfileportal.yaml:9]: Think this is a typo should be "rich" instead of "reach"

File: radar/dev/flatfileportal.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  fix validation issues on the go, and import the data to our services.
+  This includes standing data, article data, etc.
```
Comment (Tasseri) [inline radar/dev/flatfileportal.yaml:11]: Suggestion (not mandatory): Also add bullet point on what we need to evaluate

File: radar/dev/flatfileportal.yaml (hunk 1 above)
Comment (vkozynskyi-extenda) [reply radar/dev/flatfileportal.yaml:9]: You're right

File: radar/dev/flatfileportal.yaml (hunk 2 above)
Comment (vkozynskyi-extenda) [reply radar/dev/flatfileportal.yaml]: will do shortly

**Comment by Tasseri** [comment]
A great add, well done!

**Pull request merged**